source/solvers/gls_navier_stokes.cc

```diff
@@ -2042,49 +2038,70 @@ GLSNavierStokesSolver<dim>::solve_system_BiCGStab(
   const bool   renewed_matrix)
 {
   TimerOutput::Scope t(this->computing_timer, "solve");
-  auto &             system_rhs          = this->system_rhs;
-  auto &             nonzero_constraints = this->nonzero_constraints;
+    unsigned int max_iter = 3;
+    int temp_precond_fill_ilu=this->simulation_parameters.linear_solver.ilu_precond_fill;
+    unsigned int iter    = 0;
+    bool         success = false;
```

Comment:
If something is const, make it const so the user see it won't change.

Reply:
Also please use another name then temp_precond_fill_ilu. Maybe current_fill_level ?